cinch/roles/jenkins_slave_container_wrapup/templates/jswarm.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -xe
-. /etc/sysconfig/jenkins_swarm.templated
+. /var/lib/jenkins/jenkins_swarm.templated
```

Comment:
If I read that correctly, then we definitely should put this config file back into /etc and munge with permissions there, as necessary, to drive the execution of that.

But we also should move the default location of the JSwarm .jar file to /opt/jswarm as well, and the shell script could go into /opt/jswarm/bin? Or do you still think /usr/local/bin is better for that?